go/internal/ocxcli/doctor_actions.go

```
@@ -0,0 +1,142 @@
+
+func doctorFixCodexRuntime() DoctorCommandResult {
+	candidates := doctorRuntimeCandidates()
+	if len(candidates) == 0 {
+		return DoctorCommandResult{Text: "No newer Codex runtime found; keeping current selection.\nSelected: codex (unknown)\n", Exit: ExitOK}
+	}
+	selected := candidates[0]
+	var newer *doctorRuntimeCandidate
+	for index := range candidates[1:] {
+		candidate := candidates[index+1]
+		if candidate.command != selected.command && doctorCompareVersions(candidate.version, selected.version) > 0 && (newer == nil || doctorCompareVersions(candidate.version, newer.version) > 0) {
+			copy := candidate
```

golangci-lint (2.13.2)

[error] variable copy has same name as predeclared identifier

(predeclared)